compose/compose.go

```diff
@@ -58,23 +60,32 @@ var (
 	composeUpRegexp  = regexp.MustCompile("(?m:docker start <- \\(u'(.*)'\\)$)")
 )
 
-const (
-	composeProjectName = "compose"
-)
-
 // Start starts a Docker Compose configuration.
+// Fixes the Docker Compose project name to a known value so existing containers can be killed.
 // If forcePull is true, it attempts do pull newer versions of the images.
 // If rmFirst is true, it attempts to kill and delete containers before starting new ones.
 func Start(dockerComposeYML string, forcePull, rmFirst bool) (*Compose, error) {
+	return startInternal(dockerComposeYML, forcePull, rmFirst, "compose")
+}
+
+// StartParallel starts a Docker Compose configuration and is suitable for concurrent usage.
+// Note that the services should not bind to localhost ports.
+func StartParallel(dockerComposeYML string, forcePull bool) (*Compose, error) {
```

Comment:
Would you mind renaming this to StartProject and letting clients pass in the project name? I'm thinking it would be more versatile if it let clients choose their strategy for generating project names. Also I'd preserve the rmFirst parameter.

`func StartProject(dockerComposeYML string, forcePull, rmFirst bool, projectName string) (*Compose, error)`